### PR TITLE
Edited message update event handler so links don't get sent to logs

### PR DESCRIPTION
**Pull request by jacobgasser**
### What does this change/add/remove?
Added condition to see if the old message was the same as the new message in the message update event.

### Why is this beneficial to the project?
Now links will not be sent to #logs whenever the metadata for the page is received and Discord adds an embed to the message.

### CODE REVIEW

**Review by LamboCreeper — CHANGES_REQUESTED** [review]
The logic should live in the event, not the app.

File: events/messageUpdate.js [hunk 1]
```diff
@@ -6,6 +6,9 @@ const Discord = app.Discord;
 
 // The code that runs when the event is executed.
 function run(oldMessage, newMessage) {
+	if (oldMessage.content == newMessage.content) {
+		return;
+	}
```
Comment (AtrumAmantus) [inline events/messageUpdate.js]: Happy path first, no fast fails
```
if (oldMessage.content != newMessage.content) {
    //...
}```

**Review by AtrumAmantus — CHANGES_REQUESTED** [review]

File: events/messageUpdate.js (hunk 1 above)
Comment (jacobgasser) [reply events/messageUpdate.js]: okie dokie

Comment (jacobgasser) [reply events/messageUpdate.js]: can I ask why?

Comment (AtrumAmantus) [reply events/messageUpdate.js]: You should only ever have one return, and it should be at the end of the function for readability.  Using fast fails is a code smell for duct tape programming.  Not nesting your conditionals lowers the ability to codesmell for overcomplexity.

**Review by LamboCreeper — APPROVED** [review]

**Review by AtrumAmantus — APPROVED** [review]

**Pull request merged**